portfolio/src/main/webapp/js/custom-script.js

```diff
@@ -63,6 +63,8 @@ function onLoad() {
   fetchSubmittedLocations().then((locationData) => {
     populateDropdown(locationData, 'user-submitted-dropdown-menu');
   });
+
+  loadChartData();
```

Comment:
nit: move loadChartData() to below loadMap() to keep consistent loading functions

Reply:
It needs to be in the onLoad() function.